src/picomsg/src/oper_data_struct.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "../../libcharon/sa/ike_sa_state.h"
+#include "../../libstrongswan/crypto/crypters/encryption_algorithm.h"
+#include "../../libstrongswan/crypto/signers/integrity_algorithm.h"
+#include <stdint.h>
+
+struct terminator_tunnel_oper_data {
+	const char *vrf_name;
+	const char *device_id;
+	enum ike_sa_state_t status;
```

Comment:
There's already a typedef for `enum ike_sa_state_t`, no need for the extra `enum` word here